packages/launchpad/src/types.ts

```diff
@@ -7,6 +7,7 @@ export interface Task {
 	badge_text?: string;
 	actionDispatch?: () => void;
 	isLaunchTask?: boolean;
+	extra_data?: any;
```

Comment:
Rather than defining this via `any`, I think we should be much more explicit about the expected types.
```suggestion
	extra_data?: {
		about_page_id?: int;
	};
```
If we want to maintain flexibility, we could specify extra_data's structure via a dedicated interface:
```ts
interface TaskExtraData {
	about_page_id?: int;
}
```